docs/developing.md

```diff
@@ -0,0 +1,58 @@
+# Frontend Shared Package
+
+The `frontend-shared` package is a library of shared utilities, components and styles common to
+multiple Hypothesis front-end applications.
+
+## Client setup
+Follow these steps to set up ``frontend-shared`` in the client repository.
+
+#### 1.Install yalc
+ Install the `yalc` tool globally if you have not already done so.
+
+```shell
+yarn global add yalc
+```
+
+_Note_: [yalc](https://www.npmjs.com/package/yalc) is the tool that we use for locally making
+changes to a shared package and testing the changes in another project.
+
+#### 2. Build & publish
+Build the package and then publish to a a local yalc repository.
+
+```shell
+# from the frontend-shared dir
+$ yarn build
+$ yalc publish
+```
+
+#### 3. Link package
+In the ``client`` repository, link to you local pushed repository
+
+```shell
+# from the client dir
+$ yalc link  @hypothesis/frontend-shared
+```
+
+This will install the built local version of your ``frontend-shared`` repository in the ``client``
+without a using symlink.
+
+_Note_: Currently, there is no support for watchers so any changes made to the ``frontend-shared``
```

Comment:
```suggestion
_Note_: Currently, there is no support for automatically rebuilding so any changes made to the ``frontend-shared``
```

I know what is meant by _watchers_ but I think this might not be obvious to everyone.